substra/cli/interface.py

```diff
@@ -447,6 +487,8 @@ def add_algo(ctx, data, output_format, config, profile, user, verbose):
               callback=load_json_from_path)
 @click.option('--in-model-key', 'in_models_keys', type=click.STRING, multiple=True,
               help='In model traintuple key.')
+@click.option('--head-model-key')
+@click.option('--trunk-model-key')
```

Comment:
These options are for the `composite_traintuple` only, not for classic `traintuples`.